packages/terra-functional-testing/src/services/wdio-terra-service.js

```diff
@@ -1,13 +1,19 @@
 /* eslint-disable class-methods-use-this */
-const axe = require('../commands/run-axe');
+const expect = require('expect');
+const { accessibility } = require('../commands/validates');
+const { toBeAccessible } = require('../commands/expect');
 
 class TerraService {
   /**
    * Service hook executed prior to test execution.
    * Initializes the Terra Service's custom commands.
    */
   before() {
-    global.browser.addCommand('axe', axe);
+    global.expect = expect;
+    global.expect.extend({ toBeAccessible });
```

Comment:
Is this configurable from the global wdio level? Lowlight-theme will need color-contrast disabled by default.

Reply:
Yes, the global axe rule overrides are already implemented in the Terra service/axe run functionality